src/hooks/screen/useChatScreenHook.ts

```diff
@@ -184,7 +213,8 @@ function useChatScreenHook(type: 'SIGNED' | 'ANONYMOUS'): UseChatScreenHook {
       }
 
       await currentChatSchema.updateChatSentStatus(localDb, response);
-      refresh('chat');
+      refreshWithId('chat', selectedChannel?.id);
+      // refresh('chat');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to lines 187-188, the `refreshWithId` function is used instead of `refresh`. Ensure that this change doesn't introduce any inconsistencies in your application state.



<!-- This is an auto-generated comment by OSS CodeRabbit -->